src/ray/raylet/worker_pool.cc

```diff
@@ -571,6 +593,15 @@ void WorkerPool::HandleJobFinished(const JobID &job_id) {
   // Currently we don't erase the job from `all_jobs_` , as a workaround for
   // https://github.com/ray-project/ray/issues/11437.
   // unfinished_jobs_.erase(job_id);
+  auto job_config = GetJobConfig(job_id);
+  RAY_CHECK(job_config);
+  if (job_config->has_runtime_env_info() &&
+      job_config->runtime_env_info().runtime_env_eager_install()) {
+    auto const &runtime_env = job_config->runtime_env_info().serialized_runtime_env();
+    if (runtime_env != "{}" && runtime_env != "") {
```

Comment:
Is it possible to move `runtime_env != "{}" && runtime_env != ""` to a separate helper function?

Reply:
Make sense.